agency/README.md

```diff
@@ -222,6 +175,23 @@ List of valid vehicle events and the resulting vehicle status if the event is su
 | `maintenance_pick_up`  | Vehicle removed from street so it can be worked on                                                   | `available`, `unavailable`                         | `removed`           |                                                                         |
 | `deregister`           | A vehicle is deregistered                                                                            | `available`, `unavailable`, `removed`, `elsewhere` | `inactive`          | A vehicle is deactivated from the fleet and unavailable.                |
 
+## Telemetry Data
+
+A standard point of vehicle telemetry. References to latitude and longitude imply coordinates encoded in the [WGS 84 (EPSG:4326)](https://en.wikipedia.org/wiki/World_Geodetic_System) standard GPS projection expressed as [Decimal Degrees](https://en.wikipedia.org/wiki/Decimal_degrees).
+
+| Field          | Type           | Required/Optional     | Field Description                                            |
+| -------------- | -------------- | --------------------- | ------------------------------------------------------------ |
+| `vehicle_id`   | UUIDv4         | Required              | ID used in [Register](#vehicle-register)                     |
+| `timestamp`    | Unix Timestamp | Required              | Date/time that event occurred. Based on GPS clock            |
+| `gps`          | Object         | Required              | Telemetry position data                                      |
+| `gps.lat`      | Double         | Required              | Latitude of the location                                     |
+| `gps.lng`      | Double         | Required              | Longitude of the location                                    |
+| `gps.altitude` | Double         | Required              | Altitude above mean sea level in meters                      |
+| `gps.heading`  | Double         | Required              | Degrees - clockwise starting at 0 degrees at true North      |
+| `gps.speed`    | Float          | Required              | Speed in meters / sec                                        |
+| `gps.accuracy` | Integer        | Required              | GPS accuracy value                                           |
+| `charge`       | Float          | Require if Applicable | Percent battery charge of vehicle, expressed between 0 and 1 |
```

Comment:
we could imagine grouping "energy" properties using an `energy` key.
E.g.
`energy.charge`,
`energy.autonomy` (the number of miles / kilometers)
...

Reply:
Let's add to future PR with some additional clarification needed based on feedback from what data providers have available.